testing/sdk_fault_domain.py

```diff
@@ -0,0 +1,56 @@
+'''Utilities related to fault domain information from cloud providers.
+
+************************************************************************
+FOR THE TIME BEING WHATEVER MODIFICATIONS ARE APPLIED TO THIS FILE
+SHOULD ALSO BE APPLIED TO ANY OTHER PARTNER REPOS
+************************************************************************
+'''
+import logging
+
+
+log = logging.getLogger(__name__)
+
+# TODO: use cloud provider library APIs to get list of regions.
+AWS_REGIONS = [
+    "us-east-2",
+    "us-west-2",
+    "us-west-2",
+    "us-west-2",
+    'ap-northeast-1',
+    'ap-northeast-2',
+    'ap-south-1',
+    'ap-southeast-1',
+    'ap-southeast-2',
+    'ca-central-1',
+    'eu-central-1',
+    'eu-west-1',
+    'eu-west-2',
+    'sa-east-1',
+    'us-east-1',
+    'us-east-2',
```

Comment:
This value exists twice in the array

Reply:
https://github.com/mesosphere/dcos-commons/pull/1855/commits/dad9aab168597626156c9c1afc2e39f966a7fd3f

Reply:
https://github.com/mesosphere/dcos-commons/pull/1855/commits/f739fd2f512e39bccf6efd7c7d92f213427919d5